js/client-rooms.js

```diff
@@ -80,20 +80,24 @@
 				app.roomsFirstOpen = 2;
 			}
 		},
+
 		renderRoomBtn: function (roomData) {
 			var id = toID(roomData.title);
-			var buf = '<div><a href="' + app.root + id + '" class="ilink"><small style="float:right">(' + Number(roomData.userCount) + ' users)</small><strong><i class="fa fa-comment-o"></i> ' + BattleLog.escapeHTML(roomData.title) + '<br /></strong><small>' + BattleLog.escapeHTML(roomData.desc || '');
+			if (!(roomData.subRooms && roomData.subRooms.length))
+				var buf = '<div class="ilink" style="margin-bottom:7px"><a style="color:inherit" href="' + app.root + id + '"><small style="float:right">(' + Number(roomData.userCount) + ' users)</small><strong><i class="fa fa-comment-o"></i> ' + BattleLog.escapeHTML(roomData.title) + '<br /></strong><small>' + BattleLog.escapeHTML(roomData.desc || '') + '</a></div>';
 			if (roomData.subRooms && roomData.subRooms.length) {
```

Comment:
This should be an `else`. But more importantly, you're duplicating a lot of code unnecessarily. The old `<div>` does its job just fine, and doesn't need special-casing.